lib/pages/chat/file_preview.dart

```diff
@@ -69,7 +72,11 @@ class _FilePreviewState<T extends BaseChannelsCubit>
                 width: double.maxFinite,
                 height: double.maxFinite,
                 fit: BoxFit.contain,
-                imageUrl: file.downloadUrl,
```

Comment:
You don't have to do this, we've already changed the story, use downloadUrl, that's why it is here 
So we need the download icon

Reply:
I don't see the change for that in the mockup and description.